python/ray/air/_internal/usage.py

```diff
@@ -115,3 +118,40 @@ def tag_scheduler(scheduler: "TrialScheduler"):
     assert isinstance(scheduler, TrialScheduler)
     scheduler_name = _find_class_name(scheduler, "ray.tune.schedulers", TUNE_SCHEDULERS)
     record_extra_usage_tag(TagKey.TUNE_SCHEDULER, scheduler_name)
+
+
+def tag_multinode_cluster_storage(
+    local_path: str, remote_path: Optional[str], sync_config: "SyncConfig"
+) -> bool:
+    import ray
+
+    if len(ray.nodes()) <= 1:
+        return False
```

Comment:
Single node can still use cloud storage right?

Though FWIW I think usage tags are currently only logged for multi-node clusters anyway.

Reply:
Makes sense, see my proposal below